### PR TITLE
chore(installer): Moved automaticProvisionMsg under features

### LINKED ISSUE #7752: Installer: Polish `automaticProvisioning` object
## Technical Details

In the discussion https://github.com/keptn/keptn/pull/7737#discussion_r873348627, we found out that the `automaticProvisioning` options are set under two different objects: `features` and `bridge`. We should re-arrange the object and have all properties under the `features.automaticProvisioning` key.

## Acceptance Criteria

- [ ] Helm charts are updated to make use of `feature.automaticProvisioning.*`

### PR DESCRIPTION
**Pull request by RealAnna**
```
features:
  automaticProvisioning:
    serviceURL: ""
    message: ""
```

### Related Issues

Closes #7752 

### How to test

the helm chart is generated with the correct substitutions, eg. bridge and shipyard env vars 
https://github.com/keptn/keptn/runs/6999271761?check_suite_focus=true#step:3:2122 
https://github.com/keptn/keptn/runs/6999271761?check_suite_focus=true#step:3:2772

### CODE REVIEW

File: installer/manifests/keptn/templates/shipyard-controller.yaml [hunk 1]
```diff
@@ -82,7 +82,7 @@ spec:
             - name: LOG_LEVEL
               value: {{ .Values.logLevel | default "info" }}
             - name: AUTOMATIC_PROVISIONING_URL
-              value: {{ (.Values.features).automaticProvisioningURL | default "" | quote }}
+              value: {{ (.Values.features).automaticProvisioning.serviceURL | default "" | quote }}
```
Comment (odubajDT) [inline installer/manifests/keptn/templates/shipyard-controller.yaml:85]: pls remove brackets

Comment (RealAnna) [reply installer/manifests/keptn/templates/shipyard-controller.yaml:85]: I was told those are needed with new var to avoid the helm update to fail

Comment (RealAnna) [reply installer/manifests/keptn/templates/shipyard-controller.yaml:85]: I am adding them everywhere 😄

Comment (odubajDT) [reply installer/manifests/keptn/templates/shipyard-controller.yaml:85]: ok, also an option :+1:

**Review by odubajDT — APPROVED** [review]

**Comment by RealAnna** [comment]
BREAKING CHANGE:  values related to automaticProvisioning options are now both set under the features object.  bridge.automaticProvisioningMsg  --> features.automaticProvisioning.message
features.automaticProvisioningURL --> features.automaticProvisioning.serviceURL

**Pull request merged**